tools/SetupFlow/DevHome.SetupFlow.Common/DevHome.SetupFlow.Common.csproj

```diff
@@ -49,6 +50,9 @@
       <GeneratePathProperty>true</GeneratePathProperty>
       <IncludeAssets>none</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.WindowsPackageManager.Configuration.OutOfProc" Version="1.7.10091-preview">

```

Comment:
Is it ok to be using preview packages?

Reply:
Yes this is ok, we've requested this package 🙂